client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -90,12 +91,16 @@ const Progress: React.FC<IProgressProps> = ({
   const [jurisdictionDetailId, setJurisdictionDetailId] = useState<
     string | null
   >(null)
+  const [
+    jurisdictionDiscrepanciesId,
+    setJurisdictionDiscrepanciesId,
+  ] = useState<string | null>(null)
 
   const ballotsOrBatches =
     auditType === 'BATCH_COMPARISON' ? 'Batches' : 'Ballots'
 
   const someJurisdictionHasDiscrepancies = Object.values(
-    discrepancyCountsQuery.data ?? {}
+    discrepancyCountsQuery.data ? discrepancyCountsQuery.data : {}
```

Comment:
For some reason eslint wouldn't allow me to commit this file because it kept giving the following:
`103:32  ✖  This experimental syntax requires enabling the parser plugin: 'nullishCoalescingOperator' (103:32)`

Reply:
You can try upgrading to `ecmaVersion: 2021` [here](https://github.com/votingworks/arlo/blob/main/client/.eslintrc.js#L43) per this post: https://stackoverflow.com/questions/76092858/how-can-i-get-eslint-to-accept-the-nullish-coalescing-assignment

Reply:
<img width="925" alt="Screenshot 2024-11-04 at 12 06 14 PM" src="https://github.com/user-attachments/assets/a01161d2-e510-4839-afdb-c5643fca3aec">

Still getting the same error from the pre-commit hook. I did some quick debugging last week and wasn't able to fix it, and it's bit a strange since this code clearly was already committed. 

I'm wondering if it makes sense to just move forward for now, given the tight timeline, and then working on fixing this as a follow up ticket sometime soon.

Reply:
Weird, yeah I don't think it's worth figuring out right now. Also we can replace the discrepancy counts query by using the discrepancies query soon anyway.